samples/net/sockets/echo_server/overlay-qemu_cortex_m3_eth.conf

```diff
@@ -1,6 +1,7 @@
 CONFIG_NET_L2_ETHERNET=y
 CONFIG_NET_QEMU_ETHERNET=y
 
+CONFIG_SRAM_SIZE=128
```

Comment:
I'm not sure here, is it possible to modify the available RAM on emulated platforms just like this, with Kconfig change? @carlescufi Who should we ask, who could know this?

Reply:
I am sure it is possible, since it solved the problem. But, indeed, changing Hardware with Kconfig feels wrong.

Reply:
That's a good question. I don't know, maybe @nashif can let us know here.

Reply:
I think we should change it here:
https://github.com/zephyrproject-rtos/zephyr/blob/main/dts/arm/ti/lm3s6965.dtsi#L19
assuming QEMU then handles this correctly.

Reply:
The problem I see is that:
https://github.com/qemu/qemu/blob/e630bc7ec9dda656a452ed28cac4d1e9ed605d71/hw/arm/stellaris.c#L965
which is then used to determine the SRAM size:
https://github.com/qemu/qemu/blob/e630bc7ec9dda656a452ed28cac4d1e9ed605d71/hw/arm/stellaris.c#L1035

which means that the QEMU instance will only have 64KB of RAM, so not sure if this is valid.

Maybe @stephanosio or @galak can weigh in here.

Reply:
It certainly does not make sense to modify `CONFIG_SRAM_SIZE` here.

As @carlescufi noted above, the SRAM size of the emulated platform is 64KiB, so arbitrarily setting it to 128KiB here is incorrect.